---
title: Welcome to CHEM576
date: 2025-1-1
category: Jekyll
layout: post
---
This is a lecture taught by Professor Zan Luthey-Schulten during Spring 2025.

## **Course Schedule**

* **Lecture Time** : 1:00 PM - 1:50 PM
* **Days** : Monday, Wednesday, and Friday (MWF)
* **Location** : **162 Noyes Laboratory**
* **Instructors** : Prof. Z. Luthey-Schulten, T. Wu
* **CRN** : 63281
* **Term** : 01/21/25 - 05/07/25
## Office hours
ZLS on Monday 5:00 pm CLSL A552, 
Tianyu Wu on Thursday 4:00 pm-5:00 pm RAL 44
## Homework due
Friday nights. 

## Contents

### [Week1](https://forxhunter.github.io/CHEM576/jekyll/2025-01-01-week1.html)

[Week1](https://forxhunter.github.io/CHEM576/jekyll/2025-01-01-week1.html)  Web Resources & Databases for Proteins, Nucleic Acids, Genomes, Subcellular Pathways
and Networks: Deep Learning Neural Networks (NN), Sequence/Structure Alignment
Algorithms, Visualization (UNIPROT, SWISSPROT, PDB, SCOP, CATH, NCBI, KEGG,
YEASTbook, BRENDA: Smith-Waterman, STAMP, VMD/MultiSeq tutorial, Biopython)

### [Week 2](https://forxhunter.github.io/CHEM576/jekyll/2025-01-02-week2.html)

[Week 2](https://forxhunter.github.io/CHEM576/jekyll/2025-01-02-week2.html) : Evolutionary Concepts in Bioinformatics: Comparisons of sequences and structures and
structure prediction (Phylogenetic Trees, MAFFT, Blast, AlphaFold3, Foldseek)

### [Week 3](https://forxhunter.github.io/CHEM576/jekyll/2025-01-02-week3.html)

[Week 3](./subsections/week3.md): Force Fields for Biomolecules and an Introduction to Molecular Dynamics Simulations
(Nobel Prize 2013, CHARMM, AMBER, NAMD2, MARTINI/GROMACS)

### [Week 4](./subsections/week4.md)

[Week 4](./subsections/week4.md): Analyzing Molecular Dynamics Simulations: Ligand Binding, Protein Interactions
( NAMD2 and MARTINI/GROMACS tutorials, radial distribution and correlation functions)

### [Week 5](./subsections/week5.md)

[Week 5](./subsections/week5.md): Simulations of Protein:RNA and Protein:DNA Complexes and their Assembly
( Ribosome biogenesis, Protein/Nucleic Interactions, Network Analysis,)

### [Week 6](./subsections/week6.md)

[Week 6](./subsections/week6.md): Introduction to Systems Biology and Steady-State Analysis of Metabolic Networks
(Breuer et al. elife 2019, CobraPy, ESCHER)

### [Week 7](./subsections/week7.md)

[Week 7](./subsections/week7.md): Subcellular Networks for DNA Replication, Transcription, and Translation in a Minimal
Cell (Stochastic gene expression, Thornburg et al. Cell 2022, Gilbert et al. FCellDevBio 2023)

### [Week 8](./subsections/week8.md)

[Week 8](./subsections/week8.md): Whole Cell Kinetic Modeling and Simulations: Coupled Stochastic Gene Expression and
Metabolism ( Reaction-Diffusion Master Equation (RDME), Lattice Microbes tutorials, Juypter
python notebook, machine learning kinetic parameters)

### [Week 9](./subsections/week9.md)

[Week 9](./subsections/week9.md): Atomistic Simulations of a Minimal Cell (NSF STC QCB tutorials MARTINI/GROMACS
2024)

> Contents provided by Jan Stevens

### [Week 10](./subsections/week10.md)

[Week 10](./subsections/week10.md): Machine Learning of Minimal Cell Trajectories and Selection of Class Computational
Projects

> Contents supported by Rong Wei in Shulei Wang's group

### Weeks 11-14

Presentations of Computational Projects
